### DISCUSSION TITLE
Motion Threshold per defined zones.

### DISCUSSION THREAD

**Discussion by Minglarn** [blakeblackshear/frigate#1707] [Ideas]
Mabee i've missed this but.. 
Is it possible to define motion threshold per zone?

I've added this in my config, but I dont know if it really works. There is no complaints from Frigate (0.9.0-rc2) so far.
`zone_street:
  motion:
    threshold: 15`
`zone_parking:
  motion:
    threshold: 25`

**Comment by blakeblackshear** [comment]
That's not valid. Next RC will error on extra keys like that. Also, this wouldn't be easy to implement.

**Reply by Minglarn** [reply]
I see... Thanks for the info...
